packages/react/src/Header/Header.tsx

```diff
@@ -30,35 +28,29 @@ export const Header = forwardRef(
     {
       appName,
       className,
-      links,
       logoBrand = 'amsterdam',
       logoLink = '/',
       logoLinkTitle = 'Ga naar de homepage',
       menu,
       ...restProps
     }: HeaderProps,
     ref: ForwardedRef<HTMLElement>,
-  ) => {
-    return (
-      <>
-        <header {...restProps} ref={ref} className={clsx('ams-header', className)}>
-          <a href={logoLink} className="ams-header__logo-link">
-            <span className="ams-visually-hidden">{logoLinkTitle}</span>
-            <Logo brand={logoBrand} />
-          </a>
-          {links && <div className="ams-header__links">{links}</div>}
-          {menu && <div className="ams-header__menu">{menu}</div>}
-          {appName && (
-            <div className="ams-header__app-name">
-              <Heading level={1} size="level-3" className="ams-header__app-name-heading">
-                {appName}
-              </Heading>
-            </div>
-          )}
-        </header>
-      </>
-    )
-  },
+  ) => (
+    <header {...restProps} ref={ref} className={clsx('ams-header', className)}>
+      <div className="ams-header__section">
+        <a className="ams-header__logo-link" href={logoLink}>
+          <span className="ams-visually-hidden">{logoLinkTitle}</span>
+          <Logo brand={logoBrand} />
+        </a>
+        {appName && (
+          <Heading className="ams-header__app-name" level={1} size="level-5">
+            {appName}
+          </Heading>
+        )}
+      </div>
+      <div className="ams-header__section">{menu}</div>
```

Comment:
This should be a `nav`, I think. The main navigation section is the Page Menu and the Mega Menu together.

Reply:
Yes, but I think it would be best to let `PageMenu` render a `nav` as its root tag. This would add that semantics to the footer menu as well.

I think the Page Menu and the Main Menu should be separate navigation sections. Screen reader users can then select which menu they want to explore.

Reply:
> I think the Page Menu and the Main Menu should be separate navigation sections

I think that's a little odd, semantically. The entire thing is our main navigation section imo, I don't really see why we should separate them. The mega menu is just a sub menu of the main menu.

It also goes against the whole '[use landmark roles sparingly](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role#accessibility_concerns)' thing.

See also these examples, although the last 2 use the `nav` tag a bit less sparingly than I would:

- https://www.gov.uk/
- https://adobe-accessibility.github.io/Accessible-Mega-Menu/
- https://design-system.w3.org/code/header-nav.html
- https://www.rvo.nl/